Tests/test_file_qoi.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from PIL import Image, QoiImagePlugin
+
+from .helper import assert_image_equal_tofile, assert_image_similar_tofile
+
+
+class TestFileQOI:
```

Comment:
We could maybe skip the class here and use plain, top-level functions?

Reply:
You're right, I've added a commit for this.